### PR TITLE
Pre commit budget

### LINKED ISSUE #14: `data/hist_fy21` folder empty
Command:
`gunicorn -b localhost:5000 govstat:app`

```bash
[2022-02-20 10:47:40 -0500] [964739] [INFO] Starting gunicorn 20.1.0
[2022-02-20 10:47:40 -0500] [964739] [INFO] Listening at: http://127.0.0.1:5000 (964739)
[2022-02-20 10:47:40 -0500] [964739] [INFO] Using worker: sync
[2022-02-20 10:47:40 -0500] [964743] [INFO] Booting worker with pid: 964743
[2022-02-20 10:47:41 -0500] [964743] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/home/acxz/venvs/govstat-venv/lib/python3.10/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/home/acxz/venvs/govstat-venv/lib/python3.10/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/home/acxz/venvs/govstat-venv/lib/python3.10/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/home/acxz/venvs/govstat-venv/lib/python3.10/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/home/acxz/venvs/govstat-venv/lib/python3.10/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/home/acxz/venvs/govstat-venv/lib/python3.10/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/acxz/venvs/govstat-venv/lib/python3.10/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/acxz/vcs/git/github/acxz/govstat/govstat.py", line 1, in <module>
    from app import app
  File "/home/acxz/vcs/git/github/acxz/govstat/app/__init__.py", line 22, in <module>
    from app import routes, models
  File "/home/acxz/vcs/git/github/acxz/govstat/app/routes.py", line 8, in <module>
    import app.Budget as Budget
  File "/home/acxz/vcs/git/github/acxz/govstat/app/Budget.py", line 13, in <module>
    EXCEL_FILES = sorted(os.listdir(EXCEL_DIR))
FileNotFoundError: [Errno 2] No such file or directory: '/home/acxz/vcs/git/github/acxz/us/data/hist_fy21'
[2022-02-20 10:47:41 -0500] [964743] [INFO] Worker exiting (pid: 964743)
[2022-02-20 10:47:42 -0500] [964739] [INFO] Shutting down: Master
[2022-02-20 10:47:42 -0500] [964739] [INFO] Reason: Worker failed to boot.
```

### PR DESCRIPTION
**Pull request by sseshan7**
3 Commits.
First 2 are related to pre-commit, and code formatting 😃. I'm a fan of the tool 🤷🏾‍♂️ 
Added pre-commit to the requirements for development. Maybe that's not necessary as users would never need that package, only developers would.
Formatted all files using black and isort.
Added `.pre-commit-config.yaml` so future commits will be automatically cleaned.

Last one adds the budget excel files in the `data/hist_fy21` folder. I am tracking them using `git-lfs` as @acxz suggested.
Changed the path that `Budget.py` looks for the excel files.
Not prepared to remove all references to `CONGRESS_PATH` just yet, as that would break my server. Ideally, once we can specify a download destination for the `usc-run` command in `congress`, I can see moving away from the `CONGRESS_PATH`.
Would close #14.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -19,5 +19,6 @@
         "pymysql",
         "requests",
         "xlrd",
+        "pre-commit",
```
Comment (acxz) [inline setup.py]: 1) pre-commit is a dev dep so I would recommend the [following change](https://github.com/cornellius-gp/gpytorch/blob/91fb9ecc431399425e8a318accb4e5191c1b069e/setup.py#L59-L60) to the setup.py and adding it to extras/dev

2) Do users need black/isort locally? If so should they be added to the extras/dev deps like precommit

**Review by acxz — CHANGES_REQUESTED** [review]
This is lovely!!!
pre-commit, black. git-lfs
makes me giddy lol

Just a couple of nits tho:

If you could add `git-lfs` to the readme under the deps section that would be great

Edit: Also I mentioned this in a previous issue, but could you also add how you are sourcing the excel files? Just a link in the readme would suffice.

**Comment by acxz** [comment]
> Not prepared to remove all references to CONGRESS_PATH just yet, as that would break my server.

Server takes priority for sure.

File: setup.py (hunk 1 above)
Comment (sseshan7) [reply setup.py]: 1. Pretty slick, will add.
2. Users don't need to install black/isort. Pre-commit pulls those packages from github and installs them in a special pre-commit location.

**Comment by acxz** [comment]
just ping me again when u ready for a re-review

**Comment by sseshan7** [comment]
Aight @acxz updated readme, re-organized some commits, added the extras/dev section. Ready for 👀

**Review by acxz — APPROVED** [review]
thanks for iterating!
LGTM!

**Pull request merged**